### ISSUE TITLE
new metric: 

### ISSUE THREAD

**Issue by GeorgLink** [chaoss/metrics#180]
@hickmanjv proposed this metric in #83.

We will revise the metric in this Google Doc: https://docs.google.com/document/d/1uMYo80oeA9Q_ky_llFunHfeUNTxAgT2tW5kWJcu4GIA/edit?usp=sharing

Once the revision is done, we'll open a new PR.

**Comment by GeorgLink** [comment]
Sorry, wrong repository
